src/test/java/duke/parser/ParserTest.java

```diff
@@ -0,0 +1,97 @@
+package duke.parser;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class ParserTest {
+
+    private TaskList tasks = new TaskList();
+    private Ui ui = new Ui();
+    private Storage storage = new Storage("data/duke.txt");
+
+    @Test
+    void parse() {
+
+        // testing exception of delete
```

Comment:
There is a better way to test exceptions. Try to find out what does `assertThrows` do?